### ISSUE TITLE
High value for WER using Citrinet1024

### ISSUE THREAD

**Issue by MelikaBahmanabadi** [NVIDIA/NeMo#4131]
Hi, thanks for your great tutorial. 
I fine-tuned Citrinet 1024 using a Persian audio data set. The WER starts from 4, and the loss value is about 500. But I did the same with Citrinet 256, the WER starts from 0.99 and after each epoch decreases. Why in Citrinet 1024 I faced a large value for WER?

**Environment overview**

- Environment location: Google Colab

- Method of NeMo install: !pip install nemo_toolkit[asr]

- NeMo version: main

**Environment details**

- OS version : "Ubuntu20.04.3 LTS"

**Comment by titu1994** [comment]
It depends model to model. The capacity of 140 m params vs 10 m params can cause starting results to be high wer but with sufficient training the larger model will always do better.

**Comment by MelikaBahmanabadi** [comment]
In your view, at least how many epochs are necessary to fine-tune Citrinet 256 and  Citrinet 1024 on the Persian data set to get good results?

**Comment by titu1994** [comment]
Depends on the amount of training data available and amount of compute that can be used. 

In terms of number of steps, we'd recommend around 10-20,000 steps to reach convergence (or at least 100 epochs, whichever is more). 

Smaller models also converge a lot slower, and take much more epochs to reach a usable wer, whereas larger models do converge faster and get better results with longer training.
